contracts/docker-compose.yml

```diff
@@ -21,7 +21,8 @@ services:
       flextesa_node_cors_origin: "*"
     command: "hangzbox start
       --add-bootstrap-account=${JON}@2_000_000_000_000 --no-daemons-for=jon
-      --add-bootstrap-account=${LAURA}@2_000_000_000_000 --no-daemons-for=laura"
+      --add-bootstrap-account=${LAURA}@2_000_000_000_000 --no-daemons-for=laura
+      --add-bootstrap-account=${MAGGY}@2_000_000_000_000 --no-daemons-for=maggy"
```

Comment:
Thanks for the feedback. The `.env.template` file was missing the variable for `MAGGY`. I also adjusted the readme to use only `docker compose up` instead of `docker compose up sandbox`.